<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>545</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonCalcular">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="profundidadeRenalNomeLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Profundidade Renal:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fatorKNomeLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Fator K:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rimDireitoCaptacaoNomeLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Rim Direito:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="rimEsquerdoCaptacaoNomeLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Rim Esquerdo:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rimDireitoCaptacaoUnidadeLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="rimEsquerdoCaptacaoUnidadeLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="rimDireitoCaptacaoLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgreen</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rimEsquerdoCaptacaoLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>238</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgreen</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="profundidadeRenalLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgray</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fatorKLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgray</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidgetRimEsquerdo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>251</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayoutRimEsquerdo">
     <item row="0" column="0">
      <widget class="QLabel" name="rimEsquerdoLabel">
       <property name="text">
        <string>Rim Esquerdo:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="rimEsquerdoTextBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="rimEsquerdoAreaLabel">
       <property name="text">
        <string>√Årea do Rim Esquerdo:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="rimEsquerdoAreaTextBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="rimEsquerdoBGLabel">
       <property name="text">
        <string>BG do Rim Esquerdo:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="rimEsquerdoBGTextBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="rimEsquerdoBGAreaLabel">
       <property name="text">
        <string>√Årea BG do Rim Esquerdo:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="rimEsquerdoBGAreaTextBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="rimEsquerdoTempoLabel">
       <property name="text">
        <string>Tempo (segundos):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="rimEsquerdoTempoTextBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidgetRimDireito">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayoutRimDireito">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="rimDireitoLabel">
       <property name="text">
        <string>Rim Direito:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="rimDireitoTextBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="rimDireitoAreaLabel">
       <property name="text">
        <string>√Årea do Rim Direito:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="rimDireitoAreaTextBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="rimDireitoBGLabel">
       <property name="text">
        <string>BG do Rim Direito:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="rimDireitoBGTextBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="rimDireitoBGAreaLabel">
       <property name="text">
        <string>√Årea BG do Rim Direito:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="rimDireitoBGAreaTextBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="rimDireitoTempoLabel">
       <property name="text">
        <string>Tempo (segundos):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="rimDireitoTempoTextBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidgetPaciente">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>11</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutPaciente">
     <property name="verticalSpacing">
      <number>-1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="idadeNomeLabel">
       <property name="text">
        <string>Idade:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="idadeTextBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="idadeUnidadeLabel">
       <property name="text">
        <string>ano(s)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pesoNomeLabel">
       <property name="text">
        <string>Peso:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pesoTextBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="pesoUnidadeLabel">
       <property name="text">
        <string>Kg</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="alturaNomeLabel">
       <property name="text">
        <string>Altura:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="alturaTextBox"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="alturaUnidadeLabel">
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidgetPadrao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayoutPadrao">
     <item row="0" column="0">
      <widget class="QLabel" name="padraoNomeLabel">
       <property name="text">
        <string>Padr√£o:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="padraoTextBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="padraoAreaNomeLabel">
       <property name="text">
        <string>√Årea do Padr√£o:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="padraoAreaTextBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="padraoBGNomeLabel">
       <property name="text">
        <string>BG Padr√£o:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="padraoBGTextBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="padraoBGAreaNomeLabel">
       <property name="text">
        <string>√Årea do BG do Padr√£o:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="padraoBGAreaTextBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="padraoTempoTextBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="padraoTempoNomeLabel">
       <property name="text">
        <string>Tempo (segundos):</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidgetSeringas">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="layoutSeringas">
     <item row="0" column="0">
      <widget class="QLabel" name="padraoSeringaCheiaLabel">
       <property name="text">
        <string>Seringa do Padr√£o Cheia:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="padraoSeringaCheiaTextBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="padraoSeringaVaziaLabel">
       <property name="text">
        <string>Seringa do Padr√£o Vazia:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="padraoSeringaVaziaTextBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pacienteSeringaCheiaLabel">
       <property name="text">
        <string>Seringa do Paciente Cheia:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pacienteSeringaCheiaTextBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="pacienteSeringaVaziaLabel">
       <property name="text">
        <string>Seringa do Paciente Vazia:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pacienteSeringaVaziaTextBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="msgText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightblue; color: red;</string>
    </property>
    <property name="locale">
     <locale language="Portuguese" country="Brazil"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="profundidadeRenalUnidadeLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
